client/blocks/login/two-factor-authentication/two-factor-content.jsx

```diff
@@ -32,6 +32,7 @@ export default function TwoFactorContent( {
 			<div>
 				{ poller }
 				<VerificationCodeForm
+					key={ twoFactorAuthType }
```

Comment:
Confirming this is the only occurrence of the `VerificationCodeForm` component 👍